app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -389,6 +389,16 @@ class PatientSummaryScreen(
   override fun contactDoctor(patientTeleconsultationInfo: PatientTeleconsultationInfo) {
     // TODO (SM): Build WhatsApp message and start implicit intent.
   }
+
+  override fun showContactDoctorButtonTextAndIcon() {
+    contactDoctorButton.text = context.getString(R.string.patientsummary_contact_doctor)
+    contactDoctorButton.setIconResource(R.drawable.ic_whatsapp)
```

Comment:
Yeah, in layout we are actually placing the progress bar on top of the button to make it look like the progress bar is for the button. To do that in fetching state we should not be showing text and icon.

Reply:
Cool, maybe a task for future improvement would be to extend the `MaterialButton` to create a custom button that would let us switch between progress and normal states so that this behaviour can be encapsulated.